cdap-common/src/main/java/io/cdap/cdap/common/internal/remote/RemoteClient.java

```diff
@@ -174,12 +174,12 @@ private HttpResponse executeNonIdempotent(HttpRequest request) throws IOExceptio
         String message;
         String jsonDetails = null;
         if ("application/json".equals(contentType)) {
-          message = String.format("Service %s is not available (%d)", discoverableServiceName,
-              responseCode);
+          message = String.format("Service %s is not available (%d)",
```

Comment:
Please make the messages consistent in two cases. One is `(%d)` the other one is `with response code (%d)`

Reply:
Done, thanks.